lizrd/scripts/grid.py

```diff
@@ -232,13 +234,8 @@
                 *runner_params,
             ]
         elif runner == MachineBackend.LOCAL:
-            subprocess_args = [
-                get_grid_entrypoint(runner),
-                "python3",
-                "-m",
-                RUNNER,
-                *runner_params,
-            ]
+            args = argparse.Namespace(**grid[0])
+            cc_train_main(None, args=args, run_in_the_same_process=True)
```

Comment:
Nie podoba mi się, że to oznacza coupling grida z konkretnym runnerem, ale nie wiem do końca jak to lepiej zrobić. Może chociaż if-ować po `RUNNER` i w zależności od wyniku albo 1. odpalać cc_train_main, 2. działać tak jak było dotąd.

Reply:
on-line ustaliliśmy z @maciek-pioro, że w przypadku MachineBackend.LOCAL powinien on najpierw wybrać odpowienią funkcję traina i ją uruchomić.